GVRf/Framework/framework/src/main/java/org/gearvrf/jassimp2/GVRJassimpModelHelper.java

```diff
@@ -3,62 +3,110 @@
 import java.io.IOException;
 import java.util.HashMap;
 import java.util.Hashtable;
+import java.util.List;
 import java.util.Map;
 
 import org.gearvrf.FutureWrapper;
 import org.gearvrf.GVRAssetLoader;
+import org.gearvrf.GVRCamera;
+import org.gearvrf.GVRCameraRig;
 import org.gearvrf.GVRContext;
+import org.gearvrf.GVRDirectLight;
 import org.gearvrf.GVRLightBase;
 import org.gearvrf.GVRMaterial;
+import org.gearvrf.GVRPerspectiveCamera;
+import org.gearvrf.GVRPointLight;
 import org.gearvrf.GVRResourceVolume;
 import org.gearvrf.GVRMaterial.GVRShaderType;
 import org.gearvrf.GVRMesh;
 import org.gearvrf.GVRPhongShader;
 import org.gearvrf.GVRRenderData;
 import org.gearvrf.GVRSceneObject;
+import org.gearvrf.GVRSpotLight;
 import org.gearvrf.ISceneObjectEvents;
 import org.gearvrf.scene_objects.GVRModelSceneObject;
+import org.joml.Matrix4f;
 import org.joml.Quaternionf;
+import org.joml.Vector3f;
 
-public class GVRJassimpSceneObject extends GVRModelSceneObject {
-    private static final String TAG = GVRJassimpSceneObject.class.getSimpleName();
-    protected AiScene scene;
-    protected GVRResourceVolume volume;
+public final class GVRJassimpModelHelper
```

Comment:
Could be moved to the org.gearvrf package and we could have one less internal class exposed as public. Just a note, we will revisit the packages some time in the future.